src/os_func.h

```diff
@@ -48,6 +48,7 @@ int funchook_vsnprintf(char *str, size_t size, const char *format, va_list ap);
 /* os_func_windows.c */
 /* no function for now */
 #else
+#include <sys/types.h>
```

Comment:
Just encountered this issue myself. This change will need to be included whether or not the rest of the PR is accepted.